lib/auth_web/controllers/auth_controller.ex

```diff
@@ -189,6 +211,19 @@ defmodule AuthWeb.AuthController do
 
     # email is blank or invalid:
     if is_nil(email) or not Fields.Validate.email(email) do
```

Comment:
We want to log _all_ attempts even if the email is invalid. 💭
Hopefully our Frontend UI will do some validation of the email address,
but if an invalid address gets through, we still want to know that an attempt was made.